phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
A phy issue - spikedetekt is no more.

I think we're doing away with the concept of .kwd files altogether. You provide your raw data file in .dat/bin/ns5/whatever, then everything is read in realtime from this file. High-pass filtered data aren't saved, they're simply processed and features extracted as needed. Waveforms are extracted, filtered and aligned in realtime every time you load a cluster. We could write a function to low-pass the data, and save it to a new .dat or NumPy array, and if it ends up being really slow (which it shouldn't?) then we could do this at SpikeDetekt-time, but my preference would be to keep the processes separate and modular, and let people extract/save LFP with customisable parameters at some later stage (normally during analysis after clustering)...